srcpkgs/gts/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'gts'
+pkgname=gts
+version=0.7.6
+revision=1
+build_style=gnu-configure
+hostmakedepends="glib-devel pkg-config"
```

Comment:
there should be a makedepends="libglib-devel"

Reply:
Thanks! couldn't figure out what I was doing wrong